flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -204,6 +205,9 @@
 
 	protected final MailboxProcessor mailboxProcessor;
 
+	@Nullable
+	private final ExecutorService channelStateUnspillingExecutor;
```

Comment:
Add a `TODO` that it should be replaced by a global TaskManager ioExecutor?

Reply:
Maybe, not quite sure whether the existing `ioExecutor` in `TaskManagerService` can be used for unspill  state purpose. Anyway, we can always introduce a dedicated executor on `TaskManager` level future.